src/Microsoft.AspNet.Mvc.Core/Filters/AuthorizationFilter.cs

```diff
@@ -1,46 +1,25 @@
 // Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
+using System.Collections.Generic;
+using System.Linq;
 using System.Threading.Tasks;
-using Microsoft.AspNet.Mvc.Core;
 using Microsoft.AspNet.Security;
 using Microsoft.Framework.DependencyInjection;
 
 namespace Microsoft.AspNet.Mvc
 {
-    public class AuthorizeAttribute : AuthorizationFilterAttribute
+    // REVIEW: figure out how to propery rationalize this with AuthorizationFilterAttribute
+    public class AuthorizationFilter : IAsyncAuthorizationFilter, IAuthorizationFilter, IOrderedFilter
```

Comment:
you don't need to implement both AsyncAuthorizationFilter and `IAuthorizationFilter` it was there for overriding the property but probably no longer necessary?